### DISCUSSION TITLE
The catalog is blocked by bread crumbs

### DISCUSSION THREAD

**Discussion by TanxiangCode** [requarks/wiki#6152] [Error / Bug Report]
How to move the directory to the right of the screen?
![image](https://user-images.githubusercontent.com/42427662/218244852-b6930ec9-aaee-43fe-a4b7-2b7027690b43.png)

**Comment by stevenfukase** [comment]
Same here

**Comment by NGPixel** [comment]
Can you try changing the table of content location to the right, save, then back to the left? (Under Admin > Theme). Does that fixes it?

**Reply by TanxiangCode** [reply]
Yes, it's resolved, and switching back to the left again, it's no longer occluded

**Reply by NGPixel** [reply]
Thanks for confirming. I'll look into a fix for the upcoming update.
